<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>390</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>460</width>
    <height>390</height>
   </size>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../pokefinder.ico</normaloff>../pokefinder.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBoxGen3">
      <property name="title">
       <string>Gen 3</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButtonStationary3">
         <property name="text">
          <string>Stationary</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButtonWild3">
         <property name="text">
          <string>Wild</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButtonGameCube">
         <property name="text">
          <string>GameCube</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="pushButtonEgg3">
         <property name="text">
          <string>Egg</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="pushButtonIDs3">
         <property name="text">
          <string>TID/SID</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBoxGen4">
      <property name="title">
       <string>Gen 4</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButtonStationary4">
         <property name="text">
          <string>Stationary</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButtonWild4">
         <property name="text">
          <string>Wild</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButtonEgg4">
         <property name="text">
          <string>Egg</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="pushButtonIDs4">
         <property name="text">
          <string>TID/SID</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QGroupBox" name="groupBoxGen5">
      <property name="title">
       <string>Gen 5</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButtonStationary5">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Stationary</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButtonWild5">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Wild</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButtonEvent5">
         <property name="text">
          <string>Event</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="pushButtonDreamRadar">
         <property name="text">
          <string>Dream Radar</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="pushButtonHiddenGrotto">
         <property name="text">
          <string>Hidden Grotto</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QPushButton" name="pushButtonEgg5">
         <property name="text">
          <string>Egg</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QPushButton" name="pushButtonIDs5">
         <property name="text">
          <string>TID/SID</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>680</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionEncounterLookup"/>
    <addaction name="actionIVCalculator"/>
    <addaction name="actionResearcher"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuGen3Tools">
    <property name="title">
     <string>Gen 3 Tools</string>
    </property>
    <widget class="QMenu" name="menuGameCube">
     <property name="title">
      <string>GameCube</string>
     </property>
     <addaction name="actionGameCubeRTC"/>
     <addaction name="actionGameCubeSeedFinder"/>
     <addaction name="actionJirachiPattern"/>
     <addaction name="actionPokeSpot"/>
    </widget>
    <addaction name="menuGameCube"/>
    <addaction name="actionIVtoPID3"/>
    <addaction name="actionPIDtoIV"/>
    <addaction name="actionProfileManager3"/>
    <addaction name="actionSeedtoTime3"/>
    <addaction name="actionSpindaPainter"/>
   </widget>
   <widget class="QMenu" name="menuGen4Tools">
    <property name="title">
     <string>Gen 4 Tools</string>
    </property>
    <addaction name="actionIVtoPID4"/>
    <addaction name="actionProfileManager4"/>
    <addaction name="actionSeedtoTime4"/>
    <addaction name="actionSIDfromChainedShiny"/>
   </widget>
   <widget class="QMenu" name="menuGen5Tools">
    <property name="title">
     <string>Gen 5 Tools</string>
    </property>
    <addaction name="actionProfileCalibrator"/>
    <addaction name="actionProfileManager5"/>
   </widget>
   <addaction name="menuGen3Tools"/>
   <addaction name="menuGen4Tools"/>
   <addaction name="menuGen5Tools"/>
   <addaction name="menuTools"/>
  </widget>
  <action name="actionResearcher">
   <property name="text">
    <string>Researcher</string>
   </property>
  </action>
  <action name="actionSeedtoTime3">
   <property name="text">
    <string>Seed to Time</string>
   </property>
  </action>
  <action name="actionJirachiPattern">
   <property name="text">
    <string>Jirachi Pattern</string>
   </property>
  </action>
  <action name="actionPokeSpot">
   <property name="text">
    <string>PokeSpot</string>
   </property>
  </action>
  <action name="actionIVtoPID3">
   <property name="text">
    <string>IV to PID</string>
   </property>
  </action>
  <action name="actionGameCubeRTC">
   <property name="text">
    <string>GameCube RTC</string>
   </property>
  </action>
  <action name="actionPIDtoIV">
   <property name="text">
    <string>PID to IV</string>
   </property>
  </action>
  <action name="actionSeedtoTime4">
   <property name="text">
    <string>Seed to Time</string>
   </property>
  </action>
  <action name="actionIVtoPID4">
   <property name="text">
    <string>IV to PID</string>
   </property>
  </action>
  <action name="actionSIDfromChainedShiny">
   <property name="text">
    <string>SID from Chained Shiny</string>
   </property>
  </action>
  <action name="actionIVCalculator">
   <property name="text">
    <string>IV Calculator</string>
   </property>
  </action>
  <action name="actionSpindaPainter">
   <property name="text">
    <string>Spinda Painter</string>
   </property>
  </action>
  <action name="actionEncounterLookup">
   <property name="text">
    <string>Encounter Lookup</string>
   </property>
  </action>
  <action name="actionGameCubeSeedFinder">
   <property name="text">
    <string>GameCube Seed Finder</string>
   </property>
  </action>
  <action name="actionProfileCalibrator">
   <property name="text">
    <string>Profile Calibrator</string>
   </property>
  </action>
  <action name="actionProfileManager3">
   <property name="text">
    <string>Profile Manager</string>
   </property>
  </action>
  <action name="actionProfileManager4">
   <property name="text">
    <string>Profile Manager</string>
   </property>
  </action>
  <action name="actionProfileManager5">
   <property name="text">
    <string>Profile Manager</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>pushButtonStationary3</tabstop>
  <tabstop>pushButtonWild3</tabstop>
  <tabstop>pushButtonGameCube</tabstop>
  <tabstop>pushButtonEgg3</tabstop>
  <tabstop>pushButtonIDs3</tabstop>
  <tabstop>pushButtonStationary4</tabstop>
  <tabstop>pushButtonWild4</tabstop>
  <tabstop>pushButtonEgg4</tabstop>
  <tabstop>pushButtonIDs4</tabstop>
  <tabstop>pushButtonStationary5</tabstop>
  <tabstop>pushButtonWild5</tabstop>
  <tabstop>pushButtonEvent5</tabstop>
  <tabstop>pushButtonDreamRadar</tabstop>
  <tabstop>pushButtonEgg5</tabstop>
  <tabstop>pushButtonIDs5</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
